arch/arm/boot/dts/qcom/qcom-msm8974-sony-xperia-rhine-togari.dts

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: GPL-2.0
+#include "qcom-msm8974-sony-xperia-rhine.dtsi"
+
+/delete-node/ &touchscreen;
+
+/ {
+	model = "Sony Xperia Z Ultra";
+	compatible = "sony,xperia-togari", "qcom,msm8974";
+	chassis-type = "handset";
+
+	gpio-keys {
+		key-camera-snapshot {
+			status = "disabled";
+		};
+
+		key-camera-focus {
+			status = "disabled";
+		};
+	};
```

Comment:
Better move these two keys away from rhine.dtsi into amami and honami (if they both have those camera key), if togari doesn't have the camera key